.github/workflows/protocol-benchmark.yml

```diff
@@ -28,6 +28,9 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: 1.22
+      - name: Prune Docker system to free up space
+        run: |
+          docker system prune -a --volumes -f
```

Comment:
Have you seen CI fail with disk space issues?

Reply:
Yea, the first PR with just the docker compose change actually triggered this, which is why I included prune